### PR TITLE
print_hip_info working with CUDA

### PR DESCRIPTION
**Pull request by garymm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +21,7 @@ bazel run //:print_hip_info
 
 ## TODO
 
-* Get code to build and run using HIP and CUDA. Currently HIP-CPU works.
-  CUDA alone used to work before I switched to using GCC from conda.
-  Should try to get that working first.
+* Support building on ARM64.
```
Comment (oliverlee) [inline README.md:24]: FYI Bootlin toolchains are only provided for x86

File: WORKSPACE.bazel [hunk 2]
```diff
@@ -1,10 +1,10 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-BAZEL_BOOTLIN_COMMIT = "4ded1105ba0e2060003e6ac2660ff57815fe0296"
+BAZEL_BOOTLIN_COMMIT = "22ee91e01bbbc0711444c49102e92010fc6d6c95"
```
Comment (oliverlee) [inline WORKSPACE.bazel:3]: Did I use the wrong commit?

**Review by oliverlee — APPROVED** [review]

**Comment by oliverlee** [comment]
```
bazel build //...
```

is now failing for me. Is it supposed to?

```sh
ERROR: /home/oliver/repos/oliverlee/gpu-deflate/BUILD.bazel:14:13: Compiling bazel-out/k8-fastbuild/bin/hip_info.cpp.cu failed: (Exit 1): nvcc failed: error executing command (from target //:hip_info_cuda) /usr/local/cuda/bin/nvcc -x cu -Xcompiler -fPIC -ccbin external/gcc_12_2_toolchain/tool_wrappers/x86-64/2022.08-1_bleeding/x86-64-linux-gnu-2022.08-1_bleeding-gcc -I . -I bazel-out/k8-fastbuild/bin -I ... (remaining 31 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
In file included from /usr/local/cuda/bin/../targets/x86_64-linux/include/cuda_runtime.h:83,
                 from <command-line>:
/usr/local/cuda/bin/../targets/x86_64-linux/include/crt/host_config.h:139:2: error: #error -- unsupported GNU version! gcc versions later than 10 are not supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.
  139 | #error -- unsupported GNU version! gcc versions later than 10 are not supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.
      |  ^~~~~
```

File: WORKSPACE.bazel (hunk 2 above)
Comment (garymm) [reply WORKSPACE.bazel:3]: I think so.

**Comment by garymm** [comment]
Disabled CUDA by default. We can debug it more on your system later if needed.

**Pull request merged**